src/app/pages/onboarding/sign-in/sign-in.tsx

```diff
@@ -73,25 +70,21 @@ export function SignIn() {
                 <Input
                   data-testid={OnboardingSelectors.SecretKeyInput}
                   name="secretKey"
+                  css={form.values.secretKey && hideInputStyle}
                   as="textarea"
                   autoCapitalize="off"
                   autoFocus
                   borderRadius="10px"
                   fontSize="16px"
                   minHeight="168px"
-                  onChange={event => {
-                    onChange(event as ChangeEvent<HTMLInputElement>, form.handleChange);
-                  }}
+                  onChange={form.handleChange}
                   onKeyDown={e => e.key === 'Enter' && form.submitForm()}
                   onPaste={onPaste}
-                  onCopy={onCopy}
                   placeholder="Paste or type your Secret Key"
                   ref={ref as any}
                   spellCheck={false}
                   style={{ resize: 'none' }}
-                  value={
-                    isKeyMasked ? form.values.secretKey.replace(/[^ ]/g, '*') : sanitizedSecretKey
-                  }
+                  value={form.values.secretKey}
```

Comment:
Can't we remove onChange and value and bind as we could other formik inputs?

Reply:
you mean `{...form.getFieldProps('secretKey')}`